tests/providers/amazon/aws/operators/test_redshift_cluster.py

```diff
@@ -127,3 +128,26 @@ def test_pause_cluster_not_called_when_cluster_is_not_available(self, mock_get_c
         )
         redshift_operator.execute(None)
         mock_get_conn.return_value.pause_cluster.assert_not_called()
+
+
+class TestDeleteClusterOperator:
```

Comment:
Could you please add one test with `wait_for_completion` set to `false` which checks that `cluster_status` is not called?

Reply:
Added here ae3c670739fd01a390192d78476056afe19798dd